### PR TITLE
Adding badges

### PR DESCRIPTION
**Pull request by josehelps**
Adds badges to README.md for the CI job [yaml-linting](https://github.com/LOLBAS-Project/LOLBAS/blob/master/.github/workflows/yaml-linting.yml) and a script (`scripts/lolbas_count.py`) to calculate the lolbas count that generates the [svg badge](https://lolbas-project.s3.us-east-2.amazonaws.com/lolbas_count.svg), right now I uploaded it to a public s3 bucket
 
It automatically updates the svg using a newly created github action workflow called: `upload_lolbas_count` .. also included creds on the project secrets.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+
+</p>
+<p align="center">
+    <a href="https://github.com/LOLBAS-Project/LOLBAS/actions/workflows/yaml-linting.yml/badge.svg?branch=master">
+        <img src="https://img.shields.io/github/workflow/status/splunk/security_content/build-and-validate/develop" /></a>
```
Comment (wietze) [inline README.md:5]: Wrong URL? Image points to Splunk repo

File: README.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+    <a href="https://github.com/splunk/security_content">
+        <img src="https://lolbas-project.s3.us-east-2.amazonaws.com/lolbas_count.svg" /></a>
```
Comment (wietze) [inline README.md:7]: Replace with 
```</a>```
once LOLBAS-Project/LOLBAS-Project.github.io#12 has been merged

File: README.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
+
+</p>
+<p align="center">
+    <a href="https://github.com/LOLBAS-Project/LOLBAS/actions/workflows/yaml-linting.yml/badge.svg?branch=master">
+        <img src="https://img.shields.io/github/workflow/status/splunk/security_content/build-and-validate/develop" /></a>
+    <a href="https://github.com/splunk/security_content">
```
Comment (wietze) [inline README.md:6]: Wrong URL?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+        <img src="https://lolbas-project.s3.us-east-2.amazonaws.com/lolbas_count.svg" /></a>
+    <a href="https://github.com/LOLBAS-Project/LOLBAS/stargazers">
+        <img src="https://img.shields.io/github/stars/splunk/security_content?style=social" /></a>
```
Comment (wietze) [inline README.md:9]: Wrong URL?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,14 @@
+
+</p>
```
Comment (wietze) [inline README.md:2]: These lines can be removed

**Review by wietze — CHANGES_REQUESTED** [review]
Hey @d1vious , I think this solution is somewhat over-engineered and can be simplified without the need for GitHub Actions, Amazon buckets, and what not.

Have a look at LOLBAS-Project/LOLBAS-Project.github.io#12 - this adds a dynamically-updating SVG with a counter of the number of LOLBAS entries, based what's on the website. Because the website is always in sync with the main repo, this should achieve the exact same you're trying to do here.

My suggestion would therefore be:
- Remove the `lolbas_count.yml` workflow, as well as thee `scripts/` folder and `pyproject.toml` file. 
- Fix the outstanding issues in `README.md`.

File: README.md (hunk 1 above)
Comment (josehelps) [reply README.md:5]: yes this was a mistake, fixing.

File: README.md (hunk 2 above)
Comment (josehelps) [reply README.md:7]: hey quick question how does https://github.com/LOLBAS-Project/LOLBAS-Project.github.io/pull/12/files#diff-455017c6da03ae820b67ffa00b9b4e2f1c7c6cda78d8553ea74c062a44babec0R19 .. specifically `{{ site.lolbas | size }}` gets calculated here? I agree though this is a much cleaner solution, will remove the script/pyproject files etc..

Comment (josehelps) [reply README.md:7]: looking at this closer, assuming this is coming from the Jekyll collections https://github.com/LOLBAS-Project/LOLBAS-Project.github.io/blob/master/_config.yml#L8-L10 right?

File: README.md (hunk 3 above)
Comment (josehelps) [reply README.md:6]: yep fixing!

File: README.md (hunk 4 above)
Comment (josehelps) [reply README.md:9]: correct :-) I will fix it, sloppy PR on my part.

File: README.md (hunk 5 above)
Comment (josehelps) [reply README.md:2]: updated the file!

**Comment by josehelps** [comment]
@wietze thank you for the awesome feedback, implemented the changes, and yes my approach was way more complex than it needed to be thank you for pointing that out. Reminded me a bit of https://giphy.com/gifs/engineering-rube-goldberg-machine-mYqaRkXyoGbcY

**Review by wietze — APPROVED** [review]

**Pull request merged**